content/docs/user-guide/assets/overview/_index.md

```diff
@@ -0,0 +1,53 @@
+
+The table below describes some of the most common asset types in O3DE.
+
+| Asset file | Description  |
+| - | - |
+| `.bmp`, `.gif`, `.jpg`, `.jpeg`, `.png`, `.tga`, `tif`, `tiff` | Image file formats most often used as textures for materials |
+| `.dds` | A specialized compressed image format most often used for normal maps and texture atlases in real-time applications |
+| `.exr` | High dynamic range images used for Image Based Lighting (IBL) and skyboxes. |
+| `.fbx` | A 3D scene file format that stores complex meshes with texture and color information, material assignments, skin weights, skeletons, animation, levels of detail, and physics colliders |
+| `.inputbindings` | Binds input from keyboards, gamepads, touchscreens, and other devices to runtime events |
+| `.prefab` | A reusable asset that contains entities or other prefabs |
+| `.stl` | A simple mesh format used in computer aided design and 3D printing |
+| `.material` | Contains properties and values that define the look of a surface for rendering |
+| `.physicsmaterial` | Contains properties and values that define the physical properties of a surface for simulation |
+| `.shader` | A program written in AZSL (O3DE's shader language) that calculates the shading of a surface |
+
+## Asset Processing
+
+Each input asset type has a default set of rules for processing that can be customized. For graphics and animation assets, O3DE provides tools such as **Scene Settings** and **Texture Settings** that you can use to create custom processing rules on an individual asset basis. With Texture Settings, for example, you can specify how a texture is used, and how it's sampled and filtered. With Scene Settings, you can specify what meshes, skeletons, and motions to process from an `.fbx` file, and how physics colliders are generated.
+
+When the asset processing rules are customized, the options are stored in an `.assetinfo` sidecar file. If Asset Processor finds an `.assetinfo` associated with an asset, the asset is processed according to the options in the `.assetinfo` file. Otherwise, Asset Processor uses default rules to process the asset.
+
+If you have many assets to process, you might find using the Texture Settings and Scene Settings tools time consuming. The `.assetinfo` sidecar file is in JSON format and you can automate the generation and modification of `.assetinfo` files with Python. It is possible, for example, to script the export process from a DCC application, so that an `.assetinfo` file is automatically generated for the `.fbx` file. Many assets in O3DE are JSON formatted: `.material`s, `.prefab`s, and `.physicsmaterial`s, for example, can all be automatically generated and modified with Python.

```

Comment:
```suggestion
If you have many assets to process, you might find using the Texture Settings and Scene Settings tools time consuming. The `.assetinfo` sidecar file is in JSON format and you can automate the generation and modification of `.assetinfo` files with Python. It is possible, for example, to script the export process from a DCC application, so that an `.assetinfo` file is automatically generated for the `.fbx` file. Many assets in O3DE are JSON formatted: `.material`, `.prefab`, and `.physicsmaterial` files, for example, can all be automatically generated and modified with Python.

```